spec/acceptance/mod_php_spec.rb

```diff
@@ -44,7 +44,7 @@ class { 'apache::mod::php': }
       describe file("#{apache_hash['mod_dir']}/php7.4.conf") do
         it { is_expected.to contain 'DirectoryIndex index.php' }
       end
-    elsif os[:family] == 'redhat' && os[:release] =~ %r{^8\.}
+    elsif os[:family] == 'redhat' && os[:release] =~ %r{(^stream|^)8\.}
```

Comment:
Hit that PHP test failure in #2233 

Fixed it with this before seeing this PR:
https://github.com/puppetlabs/puppetlabs-apache/pull/2233/commits/af8094ba75cdab5feefaeb6c07a15c98c7194770